tests/k8s/1-020_validate_notifications/03-create-app.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+- script: |
+    set -e
+
+    cat << EOF | oc apply -f -
+    apiVersion: argoproj.io/v1alpha1
+    kind: Application
+    metadata:
+      name: my-app-3
+      namespace: $NAMESPACE
+      annotations:
+        "notifications.argoproj.io/subscribe.on-created.gmail": "jdfake@email.com"
+    spec:
+      destination:
+        namespace: $NAMESPACE
+        server: https://kubernetes.default.svc
+      project: default
+      source:
+        path: argocd/my-apps/app-2
+        repoURL: https://gitlab.com/ayush-sharma/example-assets.git
```

Comment:
It may be better not to depend on a third-party repo -- for example, if the owner deletes this repo, the tests will suddenly start failing.

Reply:
done, thanks!